Backend/api/settings.py

```diff
@@ -9,6 +9,10 @@
 IP_ADDRESS = os.getenv("IP_ADDRESS")
 PORT = os.getenv("PORT")
 
+VERSION = os.getenv("VERSION", "0.0.1")
```

Comment:
Nu vreau sa avem valori default aici, toate valorile trebuie luate din .env. Probabil tot ceva de la conflicte te rog sa lasi cum e in branch-ul Backend